lib/Models/WebProcessingServiceCatalogItem.js

```diff
@@ -191,6 +192,11 @@ WebProcessingServiceCatalogItem.prototype._load = function() {
               that.terria
             );
             catalogItem.updateFromJson(jsonCatalogItem);
+            // If the server returns duplicate names, we end up having duplicate uniqueIds.
+            // To avoid this we force uniqueId to a GUID.
+            Object.defineProperty(catalogItem, "uniqueId", {
```

Comment:
You should be able to just set the `id` property rather than clobbering the `uniqueId` property.

Reply:
Ah yes! I didn't notice that bit :man_facepalming: